imgui.cpp

```diff
@@ -2603,7 +2602,7 @@ void ImGui::RenderText(ImVec2 pos, const char* text, const char* text_end, bool
     const char* text_display_end;
     if (hide_text_after_hash)
     {
-        text_display_end = FindTextDisplayEnd(text, text_end);
+        text_display_end = ImGui::FindTextDisplayEnd(text, text_end);
```

Comment:
Could you undo adding those ImGui:: prefix for functions that are already in the ImGui namespace?
LogRenderedText is pretty much the only function here that needs them.